src/lib/astro-opengraph-image.ts

```diff
@@ -0,0 +1,27 @@
+import satori, { type Font } from "satori";
+import { html } from "satori-html";
+import sharp from "sharp";
+interface ImageResponseOptions {
+  fonts?: Font[];
+}
```

Comment:
nit: Could you refactor this to a `type` instead of `interface`

Reply:
ok